lib/tasks/publishing_api.rake

```diff
@@ -251,6 +251,17 @@ namespace :publishing_api do
     end
   end
 
+  desc "Republish all statistics which have HTML attachments to the Publishing API"
+  task republish_stats_with_html_attachments: :environment do
```

Comment:
Do we need to republish the HTML attachments directly as well, or does that happen as part of republishing the publication?

Reply:
The attachments are also republished, the PublishingAPIDocumentRepublishingWorker uses the PublishingApiHtmlAttachments service listener to take care of them.
https://github.com/alphagov/whitehall/blob/2fead7d3d01ac1240d564153ace564809d7a6b72/app/workers/publishing_api_document_republishing_worker.rb#L150